### ISSUE TITLE
[MU4 Issue] Muse Sounds: Oboe has small problems in certain intervals and dynamics (tested up to a fourth down)

### ISSUE THREAD

**Issue by erinic04** [musescore/MuseScore#14773]
**Describe the bug**
The Muse Sounds oboe has small problems in certain intervals and dynamic levels. Here's some tables of what I've found (missing intervals had no problems): 

Bends into second note
| Note | pp- | p | mp | mf | f | ff+ |
| --- | --- | --- | --- | --- | --- | --- |
 | D6 to C#6 | yes | yes | no | no | no | no | 

Overlap
| Note | pp- | p | mp | mf | f | ff+ |
| --- | --- | --- | --- | --- | --- | --- |
 | A5 to G#5 | no | no | sometimes | rarely | no | no | 

Doesn't play second note
| Note | pp- | p | mp | mf | f | ff+ |
| --- | --- | --- | --- | --- | --- | --- |
| F5 to C5 | no | no | yes | yes | no | no | 

Small pause on transition
| Note | pp- | p | mp | mf | f | ff+ |
| --- | --- | --- | --- | --- | --- | --- |
| C#5 to G#4 | no | no | yes | yes | no | no |
| C#5 to A4 | yes | yes | no | no | no | no |

Pitch bend / grace note pickup on second note
| Note | pp- | p | mp | mf | f | ff+ |
| --- | --- | --- | --- | --- | --- | --- |
| D5 to A4 | no | no | no | no | yes | yes |
| D5 to C5 | no | no | no | no | yes | yes |
| D#5 to A#4 | yes | yes | no | no | yes | yes |
| D#5 to B4 | no | no | no | no | yes | yes |
| E5 to B4 | yes | yes | no | no | a bit | a bit |
| E5 to D#5 | a bit | a bit | no | no | a bit | a bit |
 | F5 to D5 | a bit | a bit | yes | yes | no | no | 
 | F#5 to D#5 | no | no | no | no | yes | yes | 
 | A#5 to G5 | yes | yes | no | no | no | no | 
 | B5 to F#5 | yes | yes | yes | yes | no | no | 
 | B5 to A#5 | no | no | yes | yes | no | no | 
 | E6 to B5 | no | no | yes | yes | yes | yes | 
 | E6 to C6 | yes | yes | yes | yes | yes | yes | 
 | F6 to C6 | yes | yes | yes | yes | yes | yes | 
 | F6 to D#6 | no | no | no | no | yes | yes | 

Imperfection on transition
| Note | pp- | p | mp | mf | f | ff+ |
| --- | --- | --- | --- | --- | --- | --- |
| B4 to F#4 | no | no | a bit | a bit | yes | yes |
| B4 to G4 | no | no | no | no | a bit | a bit |
| B4 to A#4 | yes | yes | yes | yes | yes | yes |
| E5 to D#5 | a bit | a bit | no | no | a bit | a bit |
 | G#5 to D#5 | no | no | no | no | yes | yes | 
 | G#5 to E5 | no | no | no | no | yes | yes | 
 | G#5 to F5 | no | no | no | no | a bit | a bit | 
 | G#5 to F#5 | no | no | no | no | yes | yes | 
 | A5 to E5 | no | no | a bit | a bit | a bit | a bit | 
 | B5 to A5 | no | no | yes | yes | no | no | 
 | B5 to A#5 | no | no | no | no | yes | yes | 
 | C#6 to G#5 | no | no | no | no | yes | yes | 
 | C#6 to A#5 | no | no | yes | yes | yes | a bit | 
 | C#6 to B5 | yes | yes | yes | yes | yes | yes | 
 | C#6 to C6 | a bit | a bit | no | no | a bit | a bit | 
 | D6 to A5 | yes | yes | no | no | yes | yes | 
 | D6 to A#5 | yes | yes | yes | yes | a bit | a bit | 
 | D6 to B5 | yes | yes | a bit | a bit | yes | yes | 
 | D6 to C6 | no | no | yes | yes | no | no | 
 | D6 to C#6 | no | no | a bit | a bit | yes | yes | 
 | D#6 to B6 | no | no | yes | yes | yes | yes | 
 | D#6 to C6 | no | no | yes | yes | yes | yes | 

Glitch up on transition (grace note/imperfection, but more pronounced and combined a bit maybe)
| Note | pp- | p | mp | mf | f | ff+ |
| --- | --- | --- | --- | --- | --- | --- |
| C#6 to A5 | no | no | yes | yes | no | no | 

**To Reproduce**
1. Make one of the marked scenarios
2. Hear issue

**Expected behavior**
The playback should work well regardless of interval

**Screenshots**
too many lol

**Platform information**
 - OS: Windows

**Additional context**
pain
This took a while
also sorry if some of those are just key clicks lol, I'm getting a bit tired at the end of this

**Comment by SteveBlower** [comment]
Seems a similar problem happens with clarinet.  See #14060.  But I didn't investigate so thoroughly (exhaustively?).

**Comment by erinic04** [comment]
Yeah it was hard to differentiate between key clicks and whatever else for some of these

**Comment by erinic04** [comment]
I have also noticed some problem intervals on trumpet; should I report them or is the update a fix across the board?

**Comment by ptrumpet** [comment]
Will this mess (clarinet, oboe, bassoon noted so far) ever be fixed?  When the blurb about Muse Sounds Woodwinds talks about expert session players, I really giggle. Is the problem the original recordings or post production of samples?
